bode.py

```diff
@@ -119,14 +121,25 @@
     awg.setfrequency(AWG_CHANNEL, float(freq))
     time.sleep(TIMEOUT)
 
-    if not args.NORMALIZE:
```

Comment:
might i ask why you got rid of the normalize option? just curious

Reply:
It was moved below to line 134 to avoid duplicating it.